spec/services/distribution_create_service_spec.rb

```diff
@@ -0,0 +1,57 @@
+RSpec.describe DistributionCreateService, type: :service do
+  subject { DistributionCreateService }
+  describe "call" do
+    let!(:storage_location) { create(:storage_location, :with_items) }
+    let!(:distribution_params) { { organization_id: @organization.id, partner_id: @partner.id, storage_location_id: storage_location.id, line_items_attributes: { "0": { item_id: storage_location.items.first.id, quantity: 5 } } } }
```

Comment:
Can we use `let` here instead of `let!` since we only need this object when we use it?